app/views/static/privacy.html.erb

```diff
@@ -0,0 +1,132 @@
+<%= page_title 'Privacy notice' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">GOV.UK Verify Manage certificates privacy notice</h1>
```

Comment:
Does any of this need to be localised?

Reply:
nah, this is a static blob of text... there's no value in localizing it